cockatrice/src/gamesmodel.cpp

```diff
@@ -365,7 +349,7 @@ void GamesProxyModel::resetFilterParameters()
     gameTypeFilter.clear();
     maxPlayersFilterMin = DEFAULT_MAX_PLAYERS_MIN;
     maxPlayersFilterMax = DEFAULT_MAX_PLAYERS_MAX;
-    maxGameAgeSeconds = DEFAULT_MAX_GAME_AGE_SECONDS;
+    maxGameAge = DEFAULT_MAX_GAME_AGE;
```

Comment:
Having a unit attached, especially for time, is important

Reply:
qtime comprises units, you can access them with .hour() and .minute() for example

Reply:
Oh, gotcha

Reply:
I had _TIME appended to it but it's already a QTime object so it felt a bit double